{{cookiecutter.project_name}}/tasks.py

```diff
@@ -208,12 +208,14 @@ def mypy(c: Context):
 
 @task
 def install(c: Context):
```

Comment:
Docstring edits are also in prior PR. See comments there.

Reply:
fixed in the other PR (will merge in without it)